internal/client/policy/policy_resource.go

```diff
@@ -0,0 +1,263 @@
+	queryParams := url.Values{}
+
+	if fn.ManagementClusterName != "" {
+		queryParams["fullName.managementClusterName"] = []string{fn.ManagementClusterName}
+	}
+
+	if fn.ProvisionerName != "" {
+		queryParams["fullName.provisionerName"] = []string{fn.ProvisionerName}
+	}
+
+	requestURL := fmt.Sprintf("%s/%s/%s/%s?%s", "v1alpha1/clusters", fn.ClusterName, "policies", fn.Name, queryParams.Encode())
+
+	return p.Delete(requestURL)
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceGet gets a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceGet(fn *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyFullName) (*policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyGetPolicyResponse, error) {
+	queryParams := url.Values{}
+
+	if fn.ManagementClusterName != "" {
+		queryParams["fullName.managementClusterName"] = []string{fn.ManagementClusterName}
+	}
+
+	if fn.ProvisionerName != "" {
+		queryParams["fullName.provisionerName"] = []string{fn.ProvisionerName}
+	}
+
+	requestURL := fmt.Sprintf("%s/%s/%s/%s?%s", "v1alpha1/clusters", fn.ClusterName, "policies", fn.Name, queryParams.Encode())
+	policyClusterResponse := &policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyGetPolicyResponse{}
+	err := p.Get(requestURL, policyClusterResponse)
+
+	return policyClusterResponse, err
+}
+
+/*
+ManageV1alpha1ClusterPolicyResourceServiceUpdate updates overwrite a policy.
+*/
+func (p *Client) ManageV1alpha1ClusterPolicyResourceServiceUpdate(request *policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyRequest) (*policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyResponse, error) {
+	requestURL := fmt.Sprintf("%s/%s/%s/%s", "v1alpha1/clusters", request.Policy.FullName.ClusterName, "policies", request.Policy.FullName.Name)
+	policyClusterResponse := &policyclustermodel.VmwareTanzuManageV1alpha1ClusterPolicyPolicyResponse{}
+	err := p.Update(requestURL, request, policyClusterResponse)
+
+	return policyClusterResponse, err
+}
+
+/*
+ManageV1alpha1ClustergroupPolicyResourceServiceCreate creates a policy.
+*/
+func (p *Client) ManageV1alpha1ClustergroupPolicyResourceServiceCreate(request *policyclustergroupmodel.VmwareTanzuManageV1alpha1ClustergroupPolicyPolicyRequest) (*policyclustergroupmodel.VmwareTanzuManageV1alpha1ClustergroupPolicyPolicyResponse, error) {
+	requestURL := fmt.Sprintf("%s/%s/%s", "v1alpha1/clustergroups", request.Policy.FullName.ClusterGroupName, "policies")
+	policyClusterGroupResponse := &policyclustergroupmodel.VmwareTanzuManageV1alpha1ClustergroupPolicyPolicyResponse{}
+	err := p.Create(requestURL, request, policyClusterGroupResponse)
+
+	return policyClusterGroupResponse, err
+}
+
+/*
+ManageV1alpha1ClustergroupPolicyResourceServiceDelete deletes a policy.
+*/
+func (p *Client) ManageV1alpha1ClustergroupPolicyResourceServiceDelete(fn *policyclustergroupmodel.VmwareTanzuManageV1alpha1ClustergroupPolicyFullName) error {
```

Comment:
OrgID is required as query parameter. How are we planning to have the same?
Please check at other places as well.

Reply:
As per the generated code org id is not set in the query parameter.